packages/orchestration/scripts/fetch-chain-info.ts

```diff
@@ -26,16 +26,48 @@ const chainNames = [
   'stargaze',
   'stride',
   'umee',
+]);
+
+const nonIcaChainNames = [
+  'migaloo',
+  'terra',
+  'planq',
+  'coreum',
+  'haqq',
+  'evmos',
+  'pryzm',
+  'nibiru',
+  'carbon',
+  'sei',
+  'archway',
+  'doravota',
+  'persistence',
+  'injective',
+  'sifchain',
+  'quicksilver',
+  'provenance',
+  'crescent',
+  'nolus',
+  'empowerchain',
+  'kava',
+  'kujira',
+  'shido',
+  'dymension',
+  'lava',
+  'titan',
+  'beezee',
 ];
 
 const client = new ChainRegistryClient({
-  chainNames,
+  chainNames: [...icaChainNames, ...nonIcaChainNames],
 });
 
 // chain info, assets and ibc data will be downloaded dynamically by invoking fetchUrls method
 await client.fetchUrls();
 
-const chainInfo = await convertChainInfo(client);
+const chainInfo = await convertChainInfo(client, name =>
+  icaChainNames.has(name),
```

Comment:
Can we leave this part out for now? We should add `icaEnabled` via `withChainCapabilities` instead.

Reply:
Done